lib/helpers/is-interactive-element.js

```diff
@@ -14,23 +14,32 @@ const INTERACTIVE_TAG_NAMES = [
   'textarea',
 ];
 
-// Spec: https://www.w3.org/TR/wai-aria/complete#widget_roles
+// Spec: https://www.w3.org/TR/wai-aria-1.1/#widget_roles
 
 const ARIA_WIDGET_ROLES = [
   'button',
   'checkbox',
+  'combobox',
+  'grid',
```

Comment:
```suggestion
```
The an element with the role `grid` isn't interactive itself, but rather can contain interactive elements.